rs/execution_environment/benches/run-benchmark.sh

```diff
@@ -46,7 +47,7 @@ else
         min_result=$(echo "${min_bench}" | sed -E 's/.*bench: +([0-9]+) ns.*/\1/')
 
         if [ -z "${min_result}" ] || [ "${new_result}" -lt "${min_result}" ]; then
-            echo "    - improved ${name} time: $((new_result / 1000)) ms"
+            echo "    - improved ${name} time: $((new_result / 1000)) µs"
```

Comment:
Why the symbol was changed without changing the scaling factor?
ms (millisecond) -> 10^-3
µs (microsecond) -> 10^-6

Reply:
NIT: rename `new_result` to `new_result_ns` to make it clear why /1000 gives microseconds.